sdk/tests/conformance/extensions/00_test_list.txt

```diff
@@ -28,6 +28,7 @@ webgl-debug-shaders.html
 --min-version 1.0.3 webgl-compressed-texture-atc.html
 --min-version 1.0.3 webgl-compressed-texture-pvrtc.html
 --min-version 1.0.2 webgl-compressed-texture-s3tc.html
+--min-version 1.0.2 webgl-compressed-texture-s3tc-srgb.html
```

Comment:
--min-version 1.0.4 (top of tree version in development) at this point. This is only really for bookkeeping, but let's keep things consistent.